packages/filebrowser/src/model.ts

```diff
@@ -786,6 +787,27 @@ export class FilterFileBrowserModel extends TogglableHiddenFileBrowserModel {
     this._filterDirectories = value;
   }
 
+  /**
+   * Whether to apply fuzzy filter.
+   */
+  get useFuzzyFilter(): boolean {
+    return this._useFuzzyFilter;
+  }
+  set useFuzzyFilter(value: boolean) {
+    this._useFuzzyFilter = value;
+    this._filterFileBrowserSettingsChanged.emit({ useFuzzyFilter: value });
```

Comment:
Do we need to add this for `filterDirectories` setter too?

Reply:
Also, since this is emitting a signal indicating change, let's short-circuit if `this._useFuzzyFilter === value` already to avoid spurious re-renders :)